### DISCUSSION TITLE
[IntelliSense] trigger the IntelliSense with custom "class" or "className" in react

### DISCUSSION THREAD

**Discussion by ManuRodgers** [tailwindlabs/tailwindcss#10339] [Ideas]
![image](https://user-images.githubusercontent.com/33333844/212781531-9e14b00e-b703-4428-b4aa-7f568d25bb59.png)
Like the picture shown above, I want to ask that is it possible that VS Code tailwind IntelliSense can be triggered by some custom props such as anything starts with className***(classNameContainer in this case). Right now, as far as I know, VS Code tailwind IntelliSense can only be triggered by the prop of class or className in React.

**Comment by wongjn** [comment]
You could try using the `tailwindCSS.experimental.classRegex` setting: https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659.
